lib/perl/Genome/Sys/Lock.pm

```diff
@@ -151,30 +159,63 @@ sub with_default_lock_resource_args {
 
     $args{block_sleep} = 60 unless defined $args{block_sleep};
     $args{max_try} = 7200 unless defined $args{max_try};
+    $args{scope} = $ENV{GENOME_SYS_LOCK_DEFAULT_SCOPE} unless defined $args{scope};
```

Comment:
I agree that changing config validation does not belong in this PR.

I also think that a `genome config validate` command is a good idea, as it leaves the door open for more than environment config validation.